keyboards/idb_60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,35 @@
+#include QMK_KEYBOARD_H
+
+#define _______ KC_TRNS
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [0] = LAYOUT(
+    KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_NO,
+    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+    KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+    KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
+    KC_LCTL, KC_LGUI, KC_LALT,                                     KC_SPC,                             KC_RALT, KC_MENU, KC_RCTL
+  ),
+   [1] = LAYOUT(
+	RESET,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______,
+	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+	_______, _______, _______,                                     _______,                            _______, _______, _______
+  )
+
+};
+
+void matrix_scan_user(void) {
+
+//Layer LED indicators
+    uint32_t layer = layer_state;
+
+    if (layer & (1<<1)) {
```

Comment:
```suggestion
    if (layer_state_is(1)) {
```

No need to grab the layer state above, I think.